server/odc-service/src/main/java/com/oceanbase/odc/service/task/caller/K8sClient.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.oceanbase.odc.service.task.caller;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * @author yaobin
+ * @date 2023-11-15
+ * @since 4.2.4
+ */
+public interface K8sClient {
+
+    String createNamespaceJob(String namespace, String jobName, String image, List<String> command,
+            PodParam podParam) throws JobException;
+
+    Optional<String> getNamespaceJob(String namespace, String jobName) throws JobException;
+
+    String deleteNamespaceJob(String namespace, String jobName) throws JobException;
+}
```

Comment:
Should we hold a method to get all jobs list? such as `Collection<String> listNamespaceJobs` ?